otter/test/convergence/test_planning.py

```diff
@@ -31,13 +33,77 @@
     SetMetadataItemOnServer)
 
 
+def copy_clbdesc(clb_desc, condition=CLBNodeCondition.ENABLED, weight=1):
+    """
+    Produce a :class:`CLBDescription` from another, but with the given
```

Comment:
Just the condition is changed, not a given condition, right? That makes it sound like there's multiple conditions and you're specifying which one.